wporg-gp-translation-events.php

```diff
@@ -372,3 +373,80 @@ function () {
 		$wporg_gp_translation_events_listener->start();
 	}
 );
+
+/**
+ * Add the active events for the current user before the translation table.
+ *
+ * @return void
+ */
+function add_active_events_current_user(): void {
+	$user_attending_events      = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array();
```

Comment:
I think maybe we should consider refactoring [the code that retrieves active events the user is attending](https://github.com/WordPress/wporg-gp-translation-events/blob/trunk/includes/translation-listener.php#L69-L70) to something reusable, as I noticed we need that in several places. 

If we would do that, we would automatically get caching for the query that retrieves active events, as that can be an extensive query.

Reply:
Ok. I think it is better to address this problem in another PR, as we have an issue to it.